glotpress.php

```diff
@@ -27,7 +27,7 @@
  */
 
 define( 'GP_VERSION', '2.1.0-alpha' );
-define( 'GP_DB_VERSION', '940' );
+define( 'GP_DB_VERSION', '945' );
```

Comment:
Since #410 is going to bump this to 950 in 2.0.1, should this be 955 (or 952 to be consistent with #420)?